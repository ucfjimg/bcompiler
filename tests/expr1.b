main()
{
    extrn printf;

    printf("%d*n", 2+3);
    printf("%d*n", 2-3);
    printf("%d*n", 2*3);
    printf("%d*n", 3/2);
    printf("%d*n", 3<<2);
    printf("%d*n", 13>>2);
}
