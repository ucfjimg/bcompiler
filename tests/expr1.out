5
-1
6
1
12
3
